src/fake.ts

```diff
@@ -33,6 +43,10 @@ function iterateAllValuesFaker(dict: DictOrArray): DictOrArray {
         newDict[key] = resolveImages(f as keyof typeof faker["image"]);
         continue;
       }
+      if (k === "gender"){
+        newDict[key] = resolveGender(f);
```

Comment:
You should return random gender always instead of passing it to resolver. Just return "male" | "female" | "unset" randomly. This is how faker libs work :). Thanks for your contribution anyway! Waiting for the correction. Also please delete console.log